app/models/collection.js

```diff
@@ -28,12 +26,27 @@ class Collection {
             _.extend(this, props);
```

Comment:
If you're going with ES6 anyways, you might go with [`Object.assign`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign).